apps/price-generator/src/main/resources/application.properties

```diff
@@ -1,14 +1,16 @@
-kafka.bootstrap.servers=localhost:9092
+%prod.kafka.bootstrap.servers=localhost:9092
 
 quarkus.container-image.registry=quay.io
 quarkus.container-image.group=rhdevelopers
 quarkus.container-image.name=quarkus-tutorial-price-generator
-quarkus.container-image.tag=1.0
+quarkus.container-image.tag=2.0
 
-mp.messaging.incoming.wine.connector=smallrye-kafka
-mp.messaging.incoming.wine.topic=wine
-mp.messaging.incoming.wine.value.deserializer=org.apache.kafka.common.serialization.StringDeserializer
+mp.messaging.incoming.beer.connector=smallrye-kafka
+mp.messaging.incoming.beer.topic=beer
+mp.messaging.incoming.beer.value.deserializer=org.apache.kafka.common.serialization.StringDeserializer
 
-mp.messaging.outgoing.priced-wine.connector=smallrye-kafka
-mp.messaging.outgoing.priced-wine.topic=priced-wine
-mp.messaging.outgoing.priced-wine.value.serializer=org.apache.kafka.common.serialization.StringSerializer
\ No newline at end of file
+mp.messaging.outgoing.priced-beer.connector=smallrye-kafka
+mp.messaging.outgoing.priced-beer.topic=priced-beer
+mp.messaging.outgoing.priced-beer.value.serializer=org.apache.kafka.common.serialization.StringSerializer
+
+kafka.auto.offset.reset=earliest
```

Comment:
Why do we need this property ?

Reply:
By configuring `auto.offset.reset` to `earliest` the application binds to a topic and can consume the historical messages present in a topic. The default is set to `latest`, meaning the application receives the messages that arrived to the topic after it subscribed to the topic. I can remove it and keep the default.

Reply:
Since we are ticking every second, there will always be new records available. I'm not sure this is needed and could add confusion. `earliest` is often used when we need "replayability" behaviour

Reply:
I removed it to avoid confusion.